parachains/pallets/ping/Cargo.toml

```
@@ -30,3 +30,5 @@ std = [
 	"frame-system/std",
 	"xcm/std",
 ]
+
+try-runtime = ["frame-support/try-runtime"] 
```

`zepter lint propagate-feature --feature try-runtime -p cumulus-ping --fix` adds a few more:  

```suggestion
try-runtime = [
       "frame-support/try-runtime",
       "cumulus-pallet-xcm/try-runtime",
       "frame-system/try-runtime",
       "sp-runtime/try-runtime"
]
```

---

Cool [tool](https://github.com/ggwpez/zepter), will be sure to add this to my arsenal.